src/app.js

```diff
@@ -1 +1,26 @@
-'use strict';
+import readline from 'readline';
+import { generateRandomNumber } from './generateRandomNumber.js';
+import { calculateBullsAndCows } from './calculateBullsAndCows.js';
+const randomNumber = generateRandomNumber();
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+console.log('Welcome to the game bulls and cows!');
+
+const playBullsAndCows = () => {
+  terminal.question('Guess the number ', (userInput) => {
+    calculateBullsAndCows(userInput, randomNumber);
+
+    if (userInput === randomNumber) {
```

Comment:
You should use strict comparison '===' to compare values, but you should convert both operands to the same type before comparison. As 'userInput' is a string and 'randomNumber' is a number, they should be converted to the same type before comparison.